src/main/java/org/opensearch/performanceanalyzer/commons/metrics/AllMetrics.java

```diff
@@ -966,6 +968,8 @@ public static class Constants {
                     "searchbp_shard_stats_cancellationCount";
             public static final String SEARCHBP_SHARD_STATS_LIMITREACHEDCOUNT =
                     "searchbp_shard_stats_limitReachedCount";
+            public static final String SEARCHBP_SHARD_STATS_COMPLETIONCOUNT =
+                    "searchbp_shard_stats_completionCount";
```

Comment:
[NIT] Rename to ```searchbp_shard_task_stats_completionCount```?

Reply:
Ok, Changing all little ambiguous var names here.